tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json

```diff
@@ -447,6 +445,8 @@
     "onnx/node/generated/test_reduce_min_empty_set",
     "onnx/node/generated/test_reduce_sum_empty_set_non_reduced_axis_zero",
     "onnx/node/generated/test_resize_downsample_scales_linear_align_corners",
+    "onnx/node/generated/test_scan_sum",
+    "onnx/node/generated/test_scan9_sum",
```

Comment:
New test failures across all backends here. Can you triage a bit?

For example:
* Grab the failure logs from the earlier CI runs
* Identify which upstream change affected those tests
* Comment on the upstream change that it caused downstream test failures
* File an issue in this repository or upstream to fix the regression

Reply:
I can triage this.

Reply:
This doesn't seem to be related to this patch, but rather due to the opset version of the model being 8 rather than the minimum of 11 for the torch-onnx-to-torch conversion for `onnx.Scan` to apply.

Reply:
https://github.com/iree-org/iree-test-suites/blob/main/onnx_ops/import_onnx_tests.py converts to version 17 offline (so all tests run with upgraded ir). We have an issue tracking getting `iree-import-onnx` (or a new tool/API) to do that automatically.

Reply:
Found failure logs: https://github.com/iree-org/iree/actions/runs/11455880041/job/31873731926#step:8:93 (preserving commit and checks history by not force pushing on PRs makes this easier)

Reply:
The only appreciable difference in the generated torch IR is that we are unrolling a `torch.prim.Loop` op during the shape refinement pipeline, although it is quite odd that this causes an issue. I'll file a torch-mlir issue to finish the TODO [here](https://github.com/llvm/torch-mlir/blob/42ba541c6887f0fa2d57c896ded219cb420591ed/lib/Dialect/Torch/Transforms/SimplifyAbstractInterpCalculationsUtils.cpp#L35).

Reply:
It ended up being easy to fix. See <https://github.com/llvm/torch-mlir/pull/3812>.

Reply:
Hi @zjgarvey, I merged your patch in Torch-MLIR, and got that included in the Torch-MLIR bump in IREE but we also need a LLVM bump in IREE, since the changes in this commit https://github.com/llvm/torch-mlir/commit/aca33f1742096e7e6cb3152be15140cf9f71e508 are dependent on changes in LLVM.